opm/output/data/Aquifer.hpp

```diff
@@ -46,14 +46,31 @@ namespace Opm { namespace data {
     struct AquiferData {
         int aquiferID;    //< One-based ID, range 1..NANAQ
         double pressure;  //< Aquifer pressure
+        double fluxRate; //< Aquifer influx rate (liquid aquifer)
+        // TODO: volume should have a better name, since meaning not clear
         double volume;    //< Produced liquid volume
         double initPressure;    //< Aquifer's initial pressure
         double datumDepth;      //< Aquifer's pressure reference depth
 
         AquiferType type;
         std::shared_ptr<FetkovichData> aquFet;
+
+        double get(const std::string& key) const
+        {
+            if ( key == "AAQR" ) {
+                return this->fluxRate;
+            } else if ( key == "AAQT" ) {
+                return this->volume;
+            } else if ( key == "AAQP" ) {
+                return this->pressure;
+            }
+            return 0.;
+        }
```

Comment:
Long term, I don't think this should be a member function.  All the data members are already public, so the "string to member" mapping could be handled elsewhere.

Reply:
Yes. I have the same impression. When I was struggling to figure out the data flow, here, I used the easiest way to get the result for testing. If it is okay to you, let us address it when we add more aquifer summary keywords next time, which is in the near future.